server/engine/getMapCard.js

```diff
@@ -0,0 +1,31 @@
+const WordRoles = require("./WordRoles");
+const shuffle = require("./shuffle");
+
+const blueAgents0 = new Array(9).fill(WordRoles.BLUE);
+const redAgents0 = new Array(8).fill(WordRoles.RED);
+const blueAgents1 = new Array(8).fill(WordRoles.BLUE);
+const redAgents1 = new Array(9).fill(WordRoles.RED);
+const innocentBystanders = new Array(7).fill(WordRoles.WHITE);
+const assassins = new Array(WordRoles.BLACK);
+const deck0 = new Array().concat(
+  blueAgents0,
+  redAgents0,
+  innocentBystanders,
+  assassins
+);
+const deck1 = new Array().concat(
+  blueAgents1,
+  redAgents1,
+  innocentBystanders,
+  assassins
+);
```

Comment:
These 2 decks are basically the same except the number of agents for each team. I think it's not necessary to explicitly make 2 decks here. My suggestion for this is to just write some logic to decide on the number of agents for each team beforehand and then set it to the deck.

Reply:
Yes, you are right. I'm being lazy here. I'll fix this.